src/tiled/mainwindow.cpp

```diff
@@ -90,6 +91,8 @@
 #include <QUndoStack>
 #include <QUndoView>
 
+#include <tuple>
```

Comment:
Not actually used, right?

Reply:
Done

Good spotting, I was going to use a tuple before I created the ``ExportDetails`` struct, it caused issues and I forgot to remove the include.